app/controllers/hyrax/admin/admin_sets_controller.rb

```diff
@@ -2,9 +2,13 @@ module Hyrax
   class Admin::AdminSetsController < ApplicationController
     include Hyrax::CollectionsControllerBehavior
 
+    before_action :authenticate_user!
+    load_and_authorize_resource
```

Comment:
`load_and_authorize_resource` has to happen before additional validation or `@admin_set` is `nil`